wiki/Beatmapping/HP_drain_rate/en.md

```diff
@@ -7,7 +7,7 @@ tags:
 
 # HP drain rate
 
-*For health as a gameplay mechanic, see: [Health](/wiki/Beatmapping/Health)*  
```

Comment:
i think there actually shouldnt be any whitespace between the backslash and text/Markdown syntax

should fix for all occurences